social-guidelines.md

```diff
@@ -0,0 +1,53 @@
+# GraphQL Foundation Social Media Guidelines
+
+## Objectives ##
+The objectives of GraphQL Foundation social media:
+* Advance topics and conversations that are relevant to and beneficial for the GraphQL community
+* Continue building positive affinity for GraphQL related projects among key influencers and throughout the API community
+* Drive engagement and participation in GraphQL Foundation projects
+* Drive engagement and participation in open questions and conversations about GraphQL's future
+
+## Themes and Topics ##
+Informational, engaging, and ecosystem-focused content:
+* GraphQL Foundation projects
+* GraphQL success user stories
+* Contributor / SIG blog posts
+* Highlight beneficial open source, community projects in the GraphQL ecosystem
+* GraphQL in the API Gateway ecosystem
+* GraphQL Foundation event information and deadlines (i.e. CFP)
+* Technical topics such as security, scale, clients, and architecture
+* Informational: Tutorials, editorials, news stories. Items that help connect the dots for our community
+* Thought leadership: insights, perspectives, learnings, and experiences 
+
+## Strategy ##
+* Keep messages positive and uplifting, consistent with the [values and principles of the GraphQL Foundation](https://github.com/graphql/foundation/blob/master/CODE-OF-CONDUCT.md)
+* Communicate big picture ideas vs “announcements” or news. News will be positioned less like an announcement and more around what it means.
+* Share content across social media channels that benefit the ecosystem as a whole
+* Maintain a balance of posting on GraphQL Foundation and foundation members activities/news and ecosystem-focused content 
+* Share vendor-neutral, community-sourced posts that are informational, engaging, and ecosystem-focused
+* Engage with the community through retweets and sharing of community content.
+
+## General Guidelines ##
+
+For all GraphQL Foundation social activity, we remain a neutral foundation. Examples of the type of content:
+* Owned content is GraphQL Foundation news, blogs, case studies, survey data, etc.
+* Project content is anything sourced from [GraphQL Foundation’s currently hosted projects](https://github.com/graphql), including project news, roadmap updates, new releases, performance/security updates, blogs, conferences slides/videos, etc.
+* Ecosystem content is vendor-neutral and project-impartial sourced from contributors, maintainers, ambassadors, news outlets, etc., including blog posts, news coverage, thought leadership bylines, technology demos, sketch notes, GitHub work, GraphQL-specific Meetups, etc.
+  * CNCF is not able to share anything on our channels that promotes a vendor product or directs to a company website. 
```

Comment:
copied from the CNCF one.
Isn't that a bit strict?
Maybe times a company shares an open source library they've built.
Would that be something we cannot share at all?

Reply:
I think this is a pretty important restriction, personally.

Reply:
what about an individual that shares a blog post from a company's website or publication?

Most blog posts are on something like that no?

Reply:
I see a distinction between "a company's website" vs "a company's blog", but I hear what you mean - they control both, and it's a bit nuanced. I think adding an explicit exception for blog posts might be one way of addressing this.

Reply:
I'm with on making sure it's not becoming a marketing platform for companies, but I wonder if there are more exceptions..
On the other hand, we can always start very strict and later open more.
I'm good with that.

Reply:
the idea here would be to point to the github repo or code versus something on a company website

Reply:
@caniszczyk what about blog posts and articles?